core/domain/user_id_migration.py

```diff
@@ -40,6 +41,8 @@
     exp_models.ExplorationCommitLogEntryModel,
     exp_models.ExplorationSnapshotMetadataModel]
 
+NUMBER_OF_MODELS_IN_TRANSACTION = 20
```

Comment:
Add MAX_ before this constant.

Also move the constant to within _copy_model_with_new_id_and_user_id() -- we don't want others to use this or for it to be a general thing.

Reply:
Done.